trajectory_follower_wrapper/include/trajectory_follower_wrapper/trajectory_follower_wrapper_node.hpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2024 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include <carma_planning_msgs/msg/trajectory_plan.hpp>
+#include <autoware_msgs/msg/lane.hpp>
```

Comment:
Looks like `autoware_msgs/msg/lane.hpp` can be removed as it is unused

Reply:
removed